nowcasting_dataset/data_sources/satellite/satellite_data_source.py

```diff
@@ -346,6 +361,10 @@ def open_sat_data(zarr_path: str, consolidated: bool, logger: logging.Logger) ->
     # Flip coordinates to top-left first
     data_array = data_array.reindex(x=data_array.x[::-1])
 
+    # reindex satellite to 15 mins data
+    time = [t for t in data_array.time.values if pd.Timestamp(t).minute % 15 == 0]
```

Comment:
Should this be confirgurable? So its not hardcoded to 15 minutes?

Reply:
an yea good point